lumicks/pylake/file.py

```diff
@@ -209,3 +210,34 @@ def fdcurves(self) -> Dict[str, FDCurve]:
     @property
     def markers(self) -> Dict[str, Marker]:
         return self._get_object_dictionary("Marker", Marker)
+
+    def export_h5(self, filename, compression_level=5, omit_data={}):
+        """Write a modified h5 file to disk.
+
+        When transferring data, it can be beneficial to omit some channels from the h5 file, or use a higher compression
+        ratio. In particular, high frequency channels tend to take up a lot of space, and aren't always necessary for
+        every single analysis.
+
+        Parameters
+        ----------
+        filename : str
+            Output file name.
+        compression_level : int
+            Compression level for gzip compression (default: 5).
+        omit_data : Set[str]
+            Which data sets to omit. Should be a set of h5 paths (e.g. {"Force HF/Force 1y"}).
+
+        Examples
+        --------
+        ::
+
+            from lumicks import pylake
```

Comment:
Aren't we doing `import lumicks.pylake as lk` as the new default?

Reply:
Yup! 👍